packages/app-store/alby/components/AlbyPaymentComponent.tsx

```diff
@@ -81,15 +81,15 @@ export const AlbyPaymentComponent = (props: IAlbyPaymentComponentProps) => {
           {showQRCode && (
             <>
               <div className="flex items-center justify-center gap-2">
-                <p className="text-xs">Waiting for payment...</p>
                 <Spinner className="h-4 w-4" />
+                <p className="text-xs">Waiting for payment</p>
               </div>
               <p className="text-sm">Click or scan the invoice below to pay</p>
               <Link
                 href={`lightning:${paymentRequest}`}
-                className="inline-flex items-center justify-center rounded-2xl rounded-md border border-transparent p-2
-                font-medium text-black shadow-sm hover:brightness-95 focus:outline-none focus:ring-offset-2">
-                <QRCode size={128} value={paymentRequest} />
+                className="inline-flex items-center justify-center rounded-2xl rounded-md border border-transparent p-2 bg-white
+                font-medium text-black shadow-sm hover:brightness-95 focus:outline-none focus:ring-offset-2 p-2">
+                <QRCode size={192} value={paymentRequest} />
```

Comment:
```suggestion
                <QRCode size={128} value={paymentRequest} />
```
I think, QR is fine.

Reply:
ok cool.